src/lib/routes/admin-api/project/features.ts

```diff
@@ -242,7 +382,7 @@ export default class ProjectFeaturesController extends Controller {
             FeatureToggleDTO,
             any
         >,
-        res: Response,
```

Comment:
Could use `UpdateFeatureSchema` instead of `FeatureToggleDTO` in the request, like in `createFeature`.